util/abs/provision_pe_xl_nodes.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require "./setup/helpers/abs_helper.rb"
+include AbsHelper # rubocop:disable Style/MixinUsage
+
```

Comment:
Assuming this is the user interface.... needs some documentation about what it is supposed to do and such.

Reply:
I have a 'TODO: add a doc to the docs folder' in the description to write up the doc once the code updates are complete.